tools/report-converter/README.md

```diff
@@ -409,7 +411,29 @@ report-converter -t coccinelle -o ./codechecker_coccinelle_reports ./coccinelle_
 
 # Store the Cocccinelle reports with CodeChecker.
 CodeChecker store ./codechecker_coccinelle_reports -n coccinelle
+```
+
+## [Smatch](https://repo.or.cz/w/smatch.git)
+[Smatch](https://repo.or.cz/w/smatch.git) is a static analysis tool for C that is used on the kernel.
+
+The recommended way of running Smatch is to redirect the output to a file and
+give this file to the report converter tool.
+
+The following example shows you how to run Smatch on kernel sources 
+and store the results found by Smatch to the CodeChecker database.
+```sh
+# Change Directory to your project
+cd path/to/linux/kernel/repository
+
+# Run Smatch 
+path/to/smatch/smatch_scripts/test_kernel.sh
```

Comment:
will the `smatch_warns.txt` created automatically by the `test_kernel.sh` script or the output of that script needs to be piped into `smatch_warns.txt`?

Reply:
Yess will do that :+1: